about.html

```diff
@@ -0,0 +1,151 @@
+          <li class="desktop-nav"><a href="#">News</a></li>
+          <li class="desktop-nav"><button>TE Campaign</button></li>
+          <li class="hamburger">
+            <img
+              src="./assets/New folder/ham.png"
+              alt="hamburger"
+              id="hamburger"
+            />
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <section class="home">
+      <h3>"Hello! Sharing world"</h3>
+      <h1>THE EXPERIENCE<br />GOSPEL CONCERT 2022</h1>
+      <hr>
+      <p id="p1b">
+        ‘The Experience’ is a free, all-night Gospel music concert featuring
+        some of the most celebrated and exceptional psalmists from Nigeria and
+        around the world. This phenomenal concert is convened annually by Pastor
+        Paul Adefarasin, the Metropolitan of all House On The Rock, Lagos,
+        Nigeria.
+      </p>
+      <p id="p2b">
+        For further information,<br />kindly email us at
+        <span>teproject@houseontherock.org.ng</span><br />HOUSE ON THE ROCK
+      </p>
+    </section>
+    <section class="bottom">
+      <div class="divider">
+        <h1>over<span>500,000</span>Voices</h1>
+      </div>
+      <div class="sum-head">
+        <h2>House on the Rocks The Experience 2022</h2>
+        <hr>
+        <p>The logo of the 2022 Experience Concert was agreed on unanimously, as it is the 16th edition</p>
+        <img src="assets/aboutLOGO.png" alt="program logo" id="sum-logo" />
+      </div>
+      <hr id="linnn">
+      <div class="sum-body">
+        <h2>See the past Experience Edition</h2>
+        <hr>
+        <p>Take a look at the last two The Experience Concerts which  took place at the same venue</p>
+        <div class="cont">
+          <div class="pic-p1">
+            <h3>2020</h3>
+            <p>The Experience 14</p>
+          </div>
+          <div class="pic-p2">
+            <h3>2021</h3>
+            <p>The Experience 15</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <footer>
+      <div class="footer-top" id="mobile1">
+        <h3>Partners</h3>
+        <hr />
+        <div class="sponsors">
+          <img src="./assets/sp1.jpg" alt="sponsor1" />
+          <img src="./assets/sp2.jpg" alt="sponsor2" />
+          <img src="./assets/sp3.png" alt="sponsor3" />
+          <img src="./assets/sp4.png" alt="sponsor4" />
+        </div>
+      </div>
+      <div class="footer-bottom" id="mobile2">
+        <h2>shyn designs</h2>
+        <div class="footer-bottoma">
+          <p>2022 Microverse Capstone Project.</p>
+          <p>Some Rights Reserved.</p>
+        </div>
+      </div>
+      <div class="desktop-only" id="desktop1">
+        <div class="footer-bottom">
+          <h2 class="wit">shyn designs</h2>
+          <div class="footer-bottoma" >
+            <p class="wit">2022 Microverse Capstone Project.</p>
+            <p class="wit">Some Rights Reserved.</p>
+          </div>
```

Comment:
- [ ] As required by the previous reviewer it should be better to use the logo in the footer instead of writing this text and style it and make sure you center the content `<h2 class="wit">shyn designs</h2>`. I suggest you use the logo that you did use in the navbar. it will make your design stick more to the [design provided by microverse](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) :+1: 
- [ ] Kindly make sure the footer is displayed as expected in the mobile version.

### Your design
![image](https://user-images.githubusercontent.com/92303891/204108838-191d49bd-0e35-4df8-9bcf-c626b60e6b23.png)


### Project design
![image](https://user-images.githubusercontent.com/92303891/204108824-2617ac9c-1282-481f-8c3c-9c0e0e14bae9.png)